index.php

```diff
@@ -40,6 +40,18 @@
         'finish_flag' => false,
     ],
 ];
+
+function tasks_counter($tasks_lists, $project_name) {
+    $number_of_tasks = 0;
```

Comment:
То же самое относится и к переменным
```suggestion
    $numberOfTasks = 0;
```

Reply:
сделал